api/test/import-data/xlsx-uploads/sourcing-data-import.spec.ts

```diff
@@ -353,4 +371,148 @@ describe('Sourcing Data import', () => {
     expect(sourcingRecords[0]).toMatchObject(new SourcingRecord());
     expect(sourcingLocation).toMatchObject(new SourcingLocation());
   }, 100000);
+
+  describe('Additional config values for missing data fallback strategy and incomplete material h3 data', () => {
+    test('When a valid file is sent to the API it should return a 201 code and the data in it should be imported (error strategy)', async () => {
```

Comment:
Is this description accurate? The test is expecting a bad request expection

Reply:
nice catch, thanks